components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml

```diff
@@ -126,32 +136,17 @@
       pedigreeTrigger.observe("click", function(event) {
         event.stop();
         document.observe("xwiki:document:saved", function () {
-          window.self.location = pedigreeTrigger.href;
```

Comment:
I'm not sure why you need .search, just .location should be enough I think?

Other than that, there must be an XWiki way to get the URL for pedigree editor without hardcoding, but I don't know it